src/components/dev-hub/filter-bar.js

```diff
@@ -82,6 +84,8 @@ export default ({ heading = 'All Articles' }) => {
                         name="product"
                         choices={products}
                         defaultText="Product"
+                        value={filterValue.products}
```

Comment:
I noticed when filtering by multiple options (language and product) that the number next to the filtered type didnt change to reflect the other filter being applied. Not sure what the right thing to do here is but it looks great!

Reply:
yeah not sure what to do about that because its coming from the stitch function. Either we can not display the number, calculate it on the frontend, or just leave it (its still sort of helpful?).

Reply:
I think it is fine to leave what we have here for now and come back to it later if need be, what do you think?